scripts/completion-tests/test-completion.sh

```diff
@@ -34,10 +34,26 @@ export -f set_shell_debug_level
 set_shell_debug_level 2
 
 BINARY_NAME=helm
-BINARY_ROOT=${ROBOT_HELM_PATH:-${SCRIPT_DIR}/../../../helm/bin}
-BINARY_PATH_DOCKER=${BINARY_ROOT}/../_dist/linux-amd64
+HELM_DIR=${SCRIPT_DIR}/../../.acceptance
+HELM_REPO=${HELM_DIR}/helm
+BINARY_ROOT=${HELM_REPO}/bin
```

Comment:
The change should continue to respect `ROBOT_HELM_PATH` if it is specified.

Reply:
Yes you are right it should consider `ROBOT_HELM_PATH`, but if you [see here](https://github.com/helm/acceptance-testing/blob/master/scripts/completion-tests/test-completion.sh#L38) `_dist/linux-amd64` this dir is explicitly species here so even if set helm path its going to search `_dist/linux-amd64` and if not found will fail the test.